custom_components/tuya_local/devices/warme_gsh_heater.yaml

```diff
@@ -0,0 +1,38 @@
+name: Warme GSH heater
+products:
+  - id: 8UGflCDtNYJw1dZs
+    name: Warme GSH heater
+primary_entity:
+  entity: climate
+  dps:
+    - id: 1
+      type: boolean
+      name: hvac_mode
+      mapping:
+        - dps_val: true
+          value: "heat"
+          icon: "mdi:radiator"
+        - dps_val: false
+          value: "off"
+          icon: "mdi:radiator-disabled"
+    - id: 2
+      name: temperature
+      type: integer
+      precision: 0
+      unit: "C"
+      range:
+        min: 12
+        max: 50
+    - id: 3
+      name: current_temperature
+      type: integer
+      readonly: true
+      unit: "C"
+    - id: 101
+      name: turn_off_screen
+      type: boolean
+      typeDefaultValue: false
```

Comment:
You included typeDefaultValue here.  Does that mean that you observed that the device sometimes does not send this?

Probably this should be a separate switch or light entity to allow user control.

Reply:
> You included typeDefaultValue here. Does that mean that you observed that the device sometimes does not send this?
No, I've no reason to include `typeDefaultValue`, I will remove it

> Probably this should be a separate switch or light entity to allow user control.
Is that a different device; or a secondary entity. It's not something I care too much about, I'll never use it. I think if it's invalid I'll remove the entity `101`?

Reply:
I meant a secondary entity.   Even if you wouldn't use it, someone else might, especially if they will use the heater in a bedroom and want to turn the display off at night with an automation.

Reply:
`secondary_entities` configuration doesn't work. Updated PR with commented out section. Errors in the logs are

```
2024-02-08 22:33:04.746 WARNING (MainThread) [homeassistant.util.async_] Detected blocking call to sleep inside the event loop by custom integration 'tuya_local' at custom_components/tuya_local/device.py, line 77: self._api = tinytuya.Device(dev_id, address, local_key), please create a bug report at https://github.com/make-all/tuya-local/issues
2024-02-08 22:33:05.048 WARNING (MainThread) [homeassistant.util.async_] Detected blocking call to sleep inside the event loop by custom integration 'tuya_local' at custom_components/tuya_local/device.py, line 77: self._api = tinytuya.Device(dev_id, address, local_key), please create a bug report at https://github.com/make-all/tuya-local/issues
2024-02-08 22:33:07.373 ERROR (MainThread) [custom_components.tuya_local.helpers.config] Error adding switch_turn_off_screen for warme_gsh_heater.yaml: 'switch'
2024-02-08 22:33:07.391 ERROR (MainThread) [homeassistant.components.switch] Error while setting up tuya_local platform for switch
Traceback (most recent call last):
File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 360, in _async_setup_platform
await asyncio.shield(task)
File "/config/custom_components/tuya_local/switch.py", line 14, in async_setup_entry
await async_tuya_setup_platform(
File "/config/custom_components/tuya_local/helpers/config.py", line 57, in async_tuya_setup_platform
raise ValueError(f"{device.name} does not support use as a {platform} device.")
ValueError: Garden Room Heater does not support use as a switch device.
```

Reply:
Sorted